src/main/java/com/xceptance/xlt/report/util/JFreeChartUtils.java

```diff
@@ -1473,6 +1517,33 @@ public static void adjustSeriesForLogarithmicAxes(final TimeSeries timeSeries)
         }
     }
 
+    /**
+     * Sets the compression quality (1 -> best quality, 0 -> highest compression) to use when creating Webp images.
+     *
+     * @param quality the compression quality
+     */
+    public static void setWebpCompressionLevel(final float quality)
```

Comment:
Please rename to setWebpCompression**Quality** to be consistent. 

Since compression is lossless in our case, the "quality" is all about compression efficiency. We need to adjust the javadoc to express that, e.g. 0 -> fastest, 1 -> best compression.

Reply:
Done